src/coaster.h

```diff
@@ -243,6 +243,9 @@ class CoasterInstance : public RideInstance {
 	void Save(Saver &svr);
 
 	void RemoveFromWorld() override;
+	void InsertIntoWorld() override {
+		// This was already done during construction – nothing left to do here
```

Comment:
Sentence needs a dot at the end.

Also, I think the code style says to use C style comment here (`/* .. */`), `//` is only after code.
I am not too fussed about it though.